DependencyInjection/Configuration.php

```diff
@@ -111,6 +111,7 @@ public function getConfigTreeBuilder()
                                         ->integerNode('port')->defaultValue(6379)->end()
                                         ->scalarNode('host')->defaultValue('localhost')->end()
                                         ->integerNode('database')->isRequired()->end()
+                                        ->integerNode('password')->defaultValue(null)->end()
```

Comment:
I will close this one as I think I messed it up a little bit. (I'm sorry, a little new to pull requests)

New one: #135 #